clients/src/main/java/org/apache/kafka/common/PartitionInfo.java

```diff
@@ -20,6 +20,7 @@
  * This is used to describe per-partition state in the MetadataResponse.
  */
 public class PartitionInfo {
+    public static final int UNKNOWN_LEADER_EPOCH = -1;
```

Comment:
I think that `PartitionInfo` is a public class so I would not add this here.

Reply:
@dajac this is removed now, thanks!